test/field/FieldDriver.test.cc

```diff
@@ -0,0 +1,227 @@
+//----------------------------------*-C++-*----------------------------------//
+// Copyright 2020 UT-Battelle, LLC, and other Celeritas developers.
+// See the top-level COPYRIGHT file for details.
+// SPDX-License-Identifier: (Apache-2.0 OR MIT)
+//---------------------------------------------------------------------------//
+//! \file FieldDriver.test.cc
+//---------------------------------------------------------------------------//
+
+#include "field/FieldDriver.hh"
+#include "field/FieldParamsPointers.hh"
+#include "field/FieldInterface.hh"
+
+#include "field/RungeKuttaStepper.hh"
+#include "field/MagField.hh"
+#include "field/MagFieldEquation.hh"
+
+#include "base/Range.hh"
+#include "base/Types.hh"
+#include "base/Constants.hh"
+
+#include "celeritas_test.hh"
+
+#ifdef CELERITAS_USE_CUDA
+#    include "FieldDriver.test.hh"
+#endif
+
+using namespace celeritas;
+using namespace celeritas_test;
+
+//---------------------------------------------------------------------------//
+// TEST HARNESS
+//---------------------------------------------------------------------------//
+
+class FieldDriverTest : public Test
+{
+  protected:
+    void SetUp() override
+    {
+        // Set values of FieldParamsPointers;
+        field_params.minimum_step          = 1.0e-5 * units::millimeter;
+        field_params.delta_chord           = 0.25 * units::millimeter;
```

Comment:
Can you keep only the assignment that change the default?

Reply:
Good point.  Removed the block for setting values of FieldParamsPointers.